Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM gliderlabs/alpine
+
+COPY requirements.txt /opt/elodie/requirements.txt
+COPY docs/requirements.txt /opt/elodie/docs/requirements.txt
+COPY elodie/tests/requirements.txt /opt/elodie/elodie/tests/requirements.txt
+WORKDIR /opt/elodie
+
+RUN apk add --update boost-python ca-certificates exiftool exiv2 make python py-pip && \
```

Comment:
This is a valid concern. We could also remove those at later steps but it can become cumbersome also.
If you could at least split a bit that would be great. :+1: